examples/set-new-validators/index.ts

```diff
@@ -0,0 +1,115 @@
+import { Chain, createPublicClient, http, Address } from 'viem';
+import { privateKeyToAccount } from 'viem/accounts';
+import { arbitrumSepolia } from 'viem/chains';
+import {
+  createRollupFetchTransactionHash,
+  createRollupPrepareTransactionReceipt,
+  rollupAdminLogicPublicActions,
+} from '@arbitrum/orbit-sdk';
+import { sanitizePrivateKey } from '@arbitrum/orbit-sdk/utils';
+import { config } from 'dotenv';
+config();
+
+if (typeof process.env.ROLLUP_ADDRESS === 'undefined') {
+  throw new Error(`Please provide the "ROLLUP_ADDRESS" environment variable`);
+}
+
+if (typeof process.env.ROLLUP_OWNER_PRIVATE_KEY === 'undefined') {
+  throw new Error(`Please provide the "ROLLUP_OWNER_PRIVATE_KEY" environment variable`);
+}
+
+if (typeof process.env.NEW_VALIDATOR_ADDRESS === 'undefined') {
+  throw new Error(`Please provide the "NEW_VALIDATOR_ADDRESS" environment variable`);
+}
+
+function getBlockExplorerUrl(chain: Chain) {
+  return chain.blockExplorers?.default.url;
+}
+
+// set the parent chain and create a public client for it
+const parentChain = arbitrumSepolia;
+const parentChainPublicClient = createPublicClient({
+  chain: parentChain,
+  transport: http(),
```

Comment:
```suggestion
  transport: http(process.env.PARENT_CHAIN_RPC),
```

they need to use infura/alchemy RPC with their key otherwise they would get a timeout issue

![image](https://github.com/OffchainLabs/arbitrum-orbit-sdk/assets/13184582/396c38e0-3b12-4c67-9e92-368212f6c11d)